README.md

```diff
@@ -106,28 +106,27 @@ To create a book you need to send this payload (in json format) below:
 * Provide a working environment with your project (eg. Heroku)
 * Application must be written in Python or Go.
 * Python
-  * Use Python >= 3.5
-  * Choose any Python web framework you want to solve the problem
-  * Use PEP-8 for code style
-  * [Python Coding Style](http://docs.python-guide.org/en/latest/writing/style/)
+    * Use Python >= 3.5
+    * Choose any Python web framework you want to solve the problem
+    * Use PEP-8 for code style
+    * [Python Coding Style](http://docs.python-guide.org/en/latest/writing/style/)
 * Go
-  * Go >= 1.10
-  * [Effective Go](https://golang.org/doc/effective_go.html)
+    * Go >= 1.10
+    * [Effective Go](https://golang.org/doc/effective_go.html)
 * Every text or code must be in English
 * Write the project documentation containing:
-  * Description;
-  * Installing (setup) and testing instructions;
+    * Description;
+    * Installing (setup) and testing instructions;
     * If you provide a [docker](https://www.docker.com/) solution for setup, ensure it works without docker too.
-  * Brief description of the work environment used to run this project (Computer/operating system, text editor/IDE, libraries, etc).
-* Provide an API documentation (in English);
-* Variables, code and strings must be all in English.
+    * Brief description of the work environment used to run this project (Computer/operating system, text editor/IDE, libraries, etc).
+* Provide API documentation (in English);
+* Variables, code, and strings must be all in English.
```

Comment:
Cara tem essa vírgula aqui mesmo :thinking: ? Se fosse em português a gente escreveria: "Variáveis, código e strings"

Reply:
Inglês é diferente. Eu acho q não é errado sem a vírgula, mas eu sempre vejo ela sendo usada nesses casos.

Reply:
Bem, já vi dos dois jeitos, voto em deixar sem a vírgula.

Reply:
Me pareceu meio estranho com a vírgula, você viu algum lugar que isso está sendo aplicado recentemente @luizdepra ? Eu não me lembro de ter visto isso nem em inglês nem em português, se houver uma regra pra isso eu desconheço.... sinceramente. Vou invocar aqui um aplicador de provas de certificação que pode nos ajudar.....

@thomshio

Reply:
Aqui explica isso: https://www.grammarly.com/blog/comma-before-and/
O nosso caso é esse primeiro, lists.

Reply:
Como a vírgula é opcional vou remover.